applications/luci-app-p910nd/root/usr/share/luci/menu.d/luci-app-p910nd.json

```diff
@@ -1,11 +1,10 @@
 {
 	"admin/services/p910nd": {
-		"title": "p910nd - Printer server",
+		"title": "p910nd Printer server",
 		"order": 60,
 		"action": {
-			"type": "cbi",
-			"path": "p910nd",
-			"post": { "cbi.submit": true }
+			"type": "view",
+			"path": "print/p910nd"
```

Comment:
Path error. "print" directory does not exist.

Reply:
True - it doesn't normally - not sure where/when those paths are created. But if you mkdir -p manually, it's fine.